semgrep/semgrep/output.py

```diff
@@ -409,7 +494,7 @@ class OutputSettings(NamedTuple):
     output_per_finding_max_lines_limit: Optional[int]
     output_per_line_max_chars_limit: Optional[int]
     json_stats: bool
-    json_time: bool
+    output_time: bool
```

Comment:
Yeah, this will potentially break downstream applications.

Technically `OutputSettings` isn't part of our officially exposed API, only `semgrep_main.invoke_semgrep`. However, we may want to come up with a better story here since external consumers seem to be running into trouble with this.

We could limit our API exposure here by making everything but `output_format` optional and setting a sensible default. That way consumers specifying `OutputSettings` don't have to explicitly set all these kwargs, and we can change them more easily. That doesn't solve this particular issue of moving to `output_time`, but it could limit the damage in the future.

cc @brendongo on thoughts here.